bpf/bpf_sock.c

```diff
@@ -384,7 +384,7 @@ static __always_inline int __sock4_xlate_fwd(struct bpf_sock_addr *ctx,
 			l7backend.address = bpf_htonl(0x7f000001);
 			l7backend.port = (__be16)svc->l7_lb_proxy_port;
 			l7backend.proto = 0;
-			l7backend.pad = 0;
+			l7backend.flags = 0;
```

Comment:
I wonder if it's better to zero out the `l7backend` struct using memset right after it's declared earlier in the function?

Reply:
`struct lb4_backend l7backend = {0};` would also work if we want to use that approach.

Reply:
AFAIK, `memset` sets padding bits to `0` as well. Will need to check